### ISSUE TITLE
Installer fails 

### ISSUE THREAD

**Issue by Drsnina** [rmacoslib/r-macos-rtools#37]
everytime I try to install it through the installer for mac catalina I have "installation fail: installer encountered an error ... contact the software manufacturer ..)

**Comment by coatless** [comment]
@Drsnina not sure what went wrong, but please try the latest release for R 4.0.0

https://github.com/rmacoslib/r-macos-rtools/releases/tag/v4.0.0

**Comment by ammadrigal** [comment]
Hello I just ran into the same issue, using the latest v4.0.0. Im in Catalina

any help?

**Comment by wesslen** [comment]
Same problem where installation fails.

Running 
```
R version 4.1.1 (2021-08-10)
Platform: aarch64-apple-darwin20 (64-bit)
Running under: macOS Monterey 12.0.1
```

**Comment by luccoffeng** [comment]
Same issue here when trying to run the v4.0.0 installer on a fresh, clean macOS Monterey (prior to installing R, and prior to downloading Xcode).

As the installer failed with the message "installation fail: installer encountered an error ... contact the software manufacturer ..)", my downloads folder contained a dmg file for xcode of a few 100 kB, which was automatically deleted as I closed the failed installer.

Next, when I looked under System update, the OS momentarily reported that there were new updates for Xcode, but after a few seconds, these disappeared from the list. Xcode is not to be detected anywhere on my system.

I suspect the Xcode installation failed for some reason. I'm now manually downloading it from the app store first (v13.1) and will try to run the r-macos-rtools v4.0.0 installer again once Xcode has been installed.
